democracy_club/templates/hermes/post_detail.html

```diff
@@ -11,6 +11,11 @@
 {% block og_title_content %}{{ post.subject }}{% endblock og_title_content %}
 {% block og_type_content %}article{% endblock og_type_content %}
 
+ <!-- Twitter -->
+{% block og_image_twitter_content %}{% thumbnail post.hero "690" as im %}{{ im.url }}{% endthumbnail %}{% endblock og_image_twitter_content %} 
```

Comment:
@michaeljcollinsuk What am I missing here? I want post hero images to render in social media previews so I added the block tag here and to `base.html`